### PR TITLE
Refactoring for task variables

### PR DESCRIPTION
**Pull request by CTI777**
Related to https://github.com/Activiti/Activiti/issues/2515
Depends on: 
https://github.com/Activiti/activiti-api/pull/100
https://github.com/Activiti/Activiti/pull/2566

### CODE REVIEW

**Comment by CTI777** [comment]
@erdemedeiros @salaboy 
We can't use createVariable and setVariables method together:
https://github.com/Activiti/activiti-cloud-runtime-bundle-service/pull/258/files#diff-e6680368dcf62db2ca30224d811f544eR29
I have commented method createVariable and leave setVariables for normal user...

**Comment by CTI777** [comment]
@erdemedeiros @salaboy
We have to remove setVariables method in RB, because it is removed from API...

File: activiti-cloud-services-runtime-bundle/activiti-cloud-services-rest-api/src/main/java/org/activiti/cloud/services/rest/api/TaskVariableAdminController.java [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+	   @RequestMapping(method = RequestMethod.GET)
+	    Resources<VariableInstanceResource> getVariables(@PathVariable String taskId);
+
+	    @RequestMapping(method = RequestMethod.POST)
+	    ResponseEntity<Void> createVariable(@PathVariable String taskId,
+	                                     	@RequestBody CreateTaskVariablePayload createTaskVariablePayload);
+	    
+	    @RequestMapping(method = RequestMethod.PUT)
+	    ResponseEntity<Void> updateVariable(@PathVariable String taskId,
```
Comment (erdemedeiros) [inline activiti-cloud-services-runtime-bundle/activiti-cloud-services-rest-api/src/main/java/org/activiti/cloud/services/rest/api/TaskVariableAdminController.java]: @salaboy I'm wondering if we should include the variable name tho the path here:
`PUT /admin/v1/tasks/{taskId}/variables/{variableName}`

Comment (salaboy) [reply activiti-cloud-services-runtime-bundle/activiti-cloud-services-rest-api/src/main/java/org/activiti/cloud/services/rest/api/TaskVariableAdminController.java]: @erdemedeiros are we doing that for other things? for processes and tasks we have the id right? I will leave it to you.. I think that it make some sense as the name in this case is the unique identifier.

Comment (erdemedeiros) [reply activiti-cloud-services-runtime-bundle/activiti-cloud-services-rest-api/src/main/java/org/activiti/cloud/services/rest/api/TaskVariableAdminController.java]: Yeah, I'll add it to keep consistency with other `PUT` methods.

**Comment by erdemedeiros** [comment]
Thanks @CTI777. I've removed setVariables from here as well.

**Review by salaboy — APPROVED** [review]

**Pull request merged**